### ISSUE TITLE
Cashtag

### ISSUE THREAD

**Issue by raisty** [mastodon/mastodon#18545]
### Pitch

Please, implement cashtag into Mastodon system. 

What is cashtag `$…`?

A cashtag is a company ticker symbol preceded by the U.S. dollar sign, e.g. $TWTR. When you click on a cashtag, you'll see other Tweets mentioning that same ticker symbol.

An alternative for this can be a custom setup in the admin area, where the admin can manage all tags for the system. That can be more flexible.

### Motivation

No doubts that each Mastodon user will be a huge fan of crypto-currencies same as me. As an administrator of the pod, I will be happy to give the visitors the opportunity to tag with `$` similar to what Twitter is doing it now.

**Comment by Gargron** [comment]
Why not just use hashtag? It is literally the same thing?

> No doubts that each Mastodon user will be a huge fan of crypto-currencies same as me. 

Also no.

**Comment by raisty** [comment]
> 

To answer you why not hashtag is a very huge topic divided into plenty categories, such as economics, stock market, analysis, predictions, etc.

I don't want to be very detailed, because this is just a technical discussion. But obviously, there is a lot of sense because Twitter decides to implement it. You can find plenty of articles and use cases on the web.

There is a beautiful explanation on [Quora](https://www.quora.com/Is-there-a-difference-in-usage-between-the-tag-and-tag-on-Twitter).

TLTR;
In short `hashtag` ≠ `cashtag`.

The second part - about the crypto fans, we don't need even to discuss. It is a personal-related thing. One can love it and see there future another one hates it as hell.

**Comment by Wuzzy2** [comment]
And which standard will you use for the ticker symbols? Which stock exchange? Will it be required to fully qualify a ticker symbol (i.e. both security + stock exchange)? What if users are (expectedly) lazy and do not fully qualify the ticker symbol?

What if the company is not publicly traded?
What if a company goes bust?
What if a ticker symbol changes due to mergers? Won't that invalidate old toots?
What if the same ticker symbol has different meanings on different stock exchanges? Does a stock exchange then take precedence? Is the NYSE "more important" than the Nasdaq?
How will users be able to "escape" the dollar sign?

I don't think the idea is good as it stands right now, there are too many open questions.

**Comment by raisty** [comment]
> And which standard will you use for the ticker symbols? Which stock exchange? Will it be required to fully qualify a ticker symbol (i.e. both security + stock exchange)? What if users are (expectedly) lazy and do not fully qualify the ticker symbol?
> 
> What if the company is not publicly traded? What if a company goes bust? What if a ticker symbol changes due to mergers? Won't that invalidate old toots? What if the same ticker symbol has different meanings on different stock exchanges? Does a stock exchange then take precedence? Is the NYSE "more important" than the Nasdaq? How will users be able to "escape" the dollar sign?
> 
> I don't think the idea is good as it stands right now, there are too many open questions.

Thank you for your feedback.

In the financial world, all that you mention is relevant, but for the social platform, I don't see any of the "stock" comments as relevant. Nobody will be (neither Twitter is) a financial advisor.

`$anyticker`,`$someticker`, `$whatever`,…

Escaping the `$` sign you can do the same way as the `#` sign.

NYSE, Nasdaq, FWB, Crypto,… -> same weight

Please, feel free to raise any argument. I will be more than happy to discuss it.

**Comment by kaniini** [comment]
On Twitter the only thing $ serves as is an alternate prefix for hashtags with arbitrary length limits.  I don’t see the value this adds.

**Comment by mstdn** [comment]
We are no big fans of crypto at all. Even stronger, recently we dropped crypto donation on mstdn.social since we don't want to mess up the planet further than it already is. There are plenty other alternatives and we gotta keep crypto and NFT (bs) away from Mastodon and the Fediverse.

**Comment by pswilde** [comment]
Mastodon, and more accurately the Fediverse in general, is NOT twitter. Let's not sully the beautiful place that exists by trying to be like the hellhole of human existence that twitter is. We're better than that.

**Comment by raisty** [comment]
> Mastodon, and more accurately the Fediverse in general, is NOT twitter. Let's not sully the beautiful place that exists by trying to be like the hellhole of human existence that twitter is. We're better than that.

100% agree! Let's improve Fediverse together.

**Comment by nightpool** [comment]
(note: Github now distinguishes between issues closed as "completed" and issues closed as "not planned". This is a very new feature, and i'm not sure it's entirely a useful one, but I've adjusted this issue's "close type" to match the correct disposition of this issue)
